spec/Checker/ProductCanBeProcessedCheckerSpec.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
+declare(strict_types=1);
+
+namespace spec\BitBag\SyliusWishlistPlugin\Checker;
+
+use BitBag\SyliusWishlistPlugin\Checker\ProductCanBeProcessedChecker;
+use BitBag\SyliusWishlistPlugin\Checker\ProductQuantityCheckerInterface;
+use BitBag\SyliusWishlistPlugin\Command\Wishlist\WishlistItem;
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\OrderBundle\Controller\AddToCartCommandInterface;
+use Sylius\Component\Core\Model\OrderItemInterface;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class ProductCanBeProcessedCheckerSpec extends ObjectBehavior
```

Comment:
Finals in all classes 🙏 and there is a missing scenario 🙃

Reply:
Done/